openwebvulndb/common/parallel.py

```diff
@@ -28,7 +28,7 @@ class ParallelWorker:
     def __init__(self, worker_count, *, loop, name="Worker", timeout_per_job=None):
         self.loop = loop
         self.name = name
-        self.queue = asyncio.Queue(loop=loop)
+        self.queue = asyncio.Queue()
```

Comment:
Otherwise, I had these kind of errors 
```
Not compatible with Python 3.10 (loop parameter for async.Queue has been removed).
```
or
```
TypeError: wait_for() got an unexpected keyword argument 'loop'
```